src/Main.java

```diff
@@ -1,5 +1,90 @@
 public class Main {
     public static void main(String[] args) {
+//task1
+        System.out.println("task1");
+        int monthSum = 0;
+        int totalSum = 2_459_000;
+        int month = 0;
+        while (monthSum <= totalSum) {
+            month++;
+            monthSum = monthSum + monthSum / 100;
+            monthSum = monthSum + 15_000;
+            System.out.println("Месяц " + month + ", сумма накоплений равна " + monthSum + " рублей");
+        }
 
+//task2
+        System.out.println("task2");
+        int number = 0;
+        while (number < 10) {
+            number++;
+            System.out.print(number + " ");
+        }
+        System.out.println();
+        for (; number > 0; number--) {
+            System.out.print(number + " ");
+        }
+
+//task3
+        System.out.println();
+        System.out.println("task3");
+        int totalPopulation = 12_000_000;
+        int birth = 17;
+        int death = 8;
+        int incrementYearPopulation = (1000 + birth - death) * 12_000 - totalPopulation;
```

Comment:
и расчёт не совсем правильный, надо просто делать приращение популяции, т.е. вот это делать в цикле:
```suggestion
        totalPopulation += (birth - death) * totalPopulation / 1_000;
```